django/cantusdb_project/main_app/models/source.py

```diff
@@ -176,13 +204,13 @@ def short_heading(self) -> str:
         if holdinst := self.holding_institution:
             if holdinst.siglum and holdinst.siglum != "XX-NN":
                 title.append(f"{holdinst.siglum}")
-            elif holdinst.is_private_collector:
-                title.append("Cantus")
+        else:
+            title.append("Cantus")
 
         tt = self.shelfmark if self.shelfmark else self.title
         title.append(tt)
 
-        if not self.full_source:
+        if self.source_completeness == self.SourceCompletenessChoices.FRAGMENT:
```

Comment:
Will this start to make the "Short" heading not-so-short?

Reply:
Idk ... I can ask Debra to confirm.

Reply:
I can imagine something like `(f)` in the short display, and `(fragment)` in the full one?